data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,835 @@
+			`	As soon as he finishes, the door opens and a number of people file out, including a well-dressed Hai who immediately stops next to you.`
+			choice
+				`	(Greet the Stationkeeper.)`
+					goto greet
+				`	(Bow to the Stationkeeper.)`
+			`	She appears flattered in a Hai sort-of way. "There is no need, Captain, my station is not operational, and this is a business operation, not an appointed position. For now, anyway."`
+			label greet
+			`	You introduce yourself. "I am Kelamursh, but you should refer to me as Stationkeeper," she says in response. "Now, if you don't mind, I believe your arrival has solved our last remaining problems, putting us in good stead to depart momentarily. If I may see your ship's accommodation we'll make final preparations and depart in an hour or so."`
+			choice
+				`	"Let's go."`
+					goto depart
+				`	"We were getting paid for this, right?"`
+			`	She blinks in surprise. "Of course, didn't they tell you? Nevermind, you can confirm the details with Heyru after I've checked your vessel."`
+			label depart
+			`	You show her to her bunks. She seems satisfied, and immediately begins calling people to get things sorted. She does not seem to be the kind for casual conversation.`
+				accept
+	on accept
+		set "hr: sandsnap project"
+		log "People" "Stationkeeper Kelamursh" `Kelamursh is a high-ranking employee in an arms manufacturing company. With the concerns on Allhome, she was entrusted with the responsibility of reactivating Sandsnap Station and taking on the role of Stationkeeper.`
+
+
+mission "HRCP: Sandsnap Mercenaries"
+	landing
+	invisible
+	source "Makerplace"
+	to offer
+		has "Hai Reveal Concern Period: Reactivating Sandsnap: accepted"
+	to complete
+		never
+	npc
+		government "Hai Merchant (Human)"
+		personality escort coward opportunistic vindictive
+		fleet
+			names "hai (human)"
+			variant
+				"Grasshopper (Ion)"
+				"Grasshopper (Tracker)"
+				"Headhunter (Hai)" 2
+				"Firebird (Hai Shields)"
+				"Firebird (Hai Weapons)"
+				"Lightning Bug (Pulse)"
+				"Corvette (Hai)"
+				"Leviathan (Hai Weapons)"
+
+
+mission "Hai Reveal Concern Period: Securing Sandsnap"
+	name "Defend convoy arriving in <system>"
+	description "Keep <system> secure against pirates for the supply convoy so that they can reach <planet> safely."
+	landing
+	source "Sandsnap"
+	to offer
+		has "hr: sandsnap project"
+	on offer
+		conversation
+			`Upon arrival the Stationkeeper requests that you stay aboard your ship in case hostiles approach the station, while she suits up and goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section, and the team disappears inside.`
```

Comment:
```suggestion
			`Upon arrival, the Stationkeeper requests that you stay aboard your ship in case hostiles approach the station. She suits up and goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section, and the team disappears inside.`
```

“She suits up and goes aboard to run the access codes and start things up”

I don’t like the use of the word up twice, but I’m not sure what else to put here. I don’t think this is a super big deal, though.

Reply:
```suggestion
			`Upon arrival the Stationkeeper suits up and requests that you stay aboard your ship in case hostiles approach the station while she goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section, and the team disappears inside.`
```